### PR TITLE
Bug/icky 442

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
Sent you comments via slack

File: assets/styles/components/atoms/SfBadge.scss [hunk 1]
```diff
@@ -28,6 +28,19 @@
...
   }
 
+  &--cart-count {
+    --badge-min-width: var(--spacer-sm);
+    --badge-min-height: var(--spacer-sm);
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(var(--spacer-2xs) * 3);
```
Comment (kibo-sushant) [inline assets/styles/components/atoms/SfBadge.scss:38]: For font, use font variables like --font-size--*

File: pages/product/index.vue [hunk 2]
```diff
@@ -457,13 +439,14 @@ export default defineComponent({
       fulfillmentOptions,
       selectedFulfillmentValue,
       handleStoreLocatorClick,
-      selectFulfillmentOption,
+      // selectFulfillmentOption,
```
Comment (kibo-sushant) [inline pages/product/index.vue:442]: remove it, if not getting used

**Comment by kibo-chandradeeptalaha** [comment]
Opening a new PR with only the ticket changes.

**Pull request closed without merging**